idflakies-maven-plugin/src/main/java/edu/illinois/cs/dt/tools/plugin/ReplayMojo.java

```diff
@@ -18,16 +17,16 @@
 import java.nio.file.Paths;
 import java.util.List;
 
-public class ReplayPlugin extends TestPlugin {
+public class ReplayMojo extends AbstractIDFlakiesMojo {
```

Comment:
Give Mojo name

Reply:
Do this for all classes that extend AbIDFMojo